pywhatkit/mail.py

```diff
@@ -34,3 +35,11 @@ def send_mail(email_sender: str, password: str, subject: str,
 
         smtp.send_message(msg)
         print('Email sent Successfully!')
+        
+        
+def send_hmail(email_sender: str, password: str, subject: str,
```

Comment:
The message parameter of send_mail accepts in a `str` type but here we are passing on a `MIMEText`, modify the send_mail function and add the following to the message parameter 
`message: Union[str, MIMEText]`. Import the Union from `from typing import Union`